jib-core/src/main/java/com/google/cloud/tools/jib/http/Authorization.java

```diff
@@ -45,23 +114,26 @@ public static Authorization fromBearerToken(String token) {
   public static Authorization fromBasicCredentials(String username, String secret) {
     String credentials = username + ":" + secret;
     String token = Base64.encodeBase64String(credentials.getBytes(StandardCharsets.UTF_8));
-    return new Authorization("Basic", token);
+    return new Authorization("Basic", token, null);
   }
 
   /**
    * @param token the token
    * @return an {@link Authorization} with a base64-encoded {@code username:password} string
    */
   public static Authorization fromBasicToken(String token) {
-    return new Authorization("Basic", token);
+    return new Authorization("Basic", token, null);
   }
 
   private final String scheme;
   private final String token;
+  @Nullable private final Multimap<String, String> repositoryGrants;
```

Comment:
Since the token and this value is related (i.e., the value is actually part of a token) if not null, I'd like to add a comment like

"If token is a Bearer token, repositoryGrants must be the repository grant information extracted from the token. Otherwise, it must be null."

or something similar you think suitable. Because when someone new sees these fields, they may think these values can be independent.